basic_system/src/system_implementation/flat_storage_model/account_cache_entry.rs

```diff
@@ -130,23 +130,39 @@ pub struct AccountProperties {
     pub nonce: u64,
     pub balance: U256,
     pub bytecode_hash: Bytes32,
-    pub bytecode_len: u32,
+    pub unpadded_code_len: u32,
```

Comment:
Did you switch from logic of full len = fn(bytecode len, artifacts) to the bytecode_len = fn(full len, artifacts)? If yes - it loses information

Reply:
Which information is lost? Now bytecode_len (full) = code_len + padding + artifacts_len